doc/api/typescript.md

```diff
@@ -68,6 +68,18 @@ By intentionally not supporting syntaxes that require JavaScript code
 generation, and by replacing inline types with whitespace, Node.js can run
 TypeScript code without the need for source maps.
 
+The suggested `tsconfig.json` settings for type stripping are:
+
+```json
+{
+    "compilerOptions": {
+        "allowImportingTsExtensions": true,
+        "rewriteRelativeImportExtensions": true,
+        "verbatimModuleSyntax": true,
```

Comment:
You all would know better than me, but does whatever renders this markdown care about the trailing comma in a `json` code fence?

Reply:
fixed anyways